frontend/src/interfaces/History.ts

```diff
@@ -0,0 +1,42 @@
+export interface IWeaponHistory{
+  buyerAddress: string
```

Comment:
all of these need semicolons. not sure why the linter isn't picking that up.

Reply:
edited the semicolons